adhocracy-plus/assets/scss/components/_button.scss

```diff
@@ -175,6 +175,26 @@
         color: $text-color;
         cursor: not-allowed;
     }
+
+    &.collapsible {
+        .link--less {
+            display: none;
+        }
+
+        .link--more {
+            display: block;
+        }
+    }
+
+    .show + &.collapsible {
+        .link--less {
+            display: block;
+        }
+
+        .link--more {
+            display: none;
+        }
+    }
 }
```

Comment:
I would add the `transition: none` here as it looks a little confusing when swapping between 2